src/coriolis.jl

```diff
@@ -24,22 +24,59 @@ end
     FPlane([T=Float64;] f)
 
 Returns a parameter object for constant rotation at the angular frequency
-`2f`, and therefore with background vorticity `f`, around a vertical axis.
+`f`, and therefore with background vorticity `2f`, around a vertical axis.
 
-Also called `FPlane`, after the "f-plane" approximation for the local effect of 
+Also called `FPlane`, after the "f-plane" approximation for the local effect of
 Earth's rotation in a planar coordinate system tangent to the Earth's surface.
 """
-function FPlane(T=Float64; f) 
+function FPlane(T=Float64; f)
     return FPlane{T}(f)
 end
 
-@inline fv(i, j, k, grid::RegularCartesianGrid{T}, f, v) where T = 
+"""
+    FPlane([T=Float64;] Ω, lat)
+
+Returns a parameter object for constant rotation at the angular frequency
+`f = 2Ωsin(lat)`, and therefore with background vorticity `f = 4Ωsin(lat)`,
```

Comment:
Isn't the angular frequency `2f`? For a fluid in solid body rotation, the vorticity is half the rotation rate / angular frequency.

Reply:
Vorticity is twice the rotation rate. I think the previous function's docstring had it wrong and I did not bother to check.